pkg/agent/cniserver/ipam/antrea_ipam.go

```diff
@@ -365,4 +366,7 @@ func init() {
 
 	// Host local plugin is fallback driver
 	RegisterIPAMDriver(AntreaIPAMType, &IPAMDelegator{pluginType: ipamHostLocal})
+
+	// Whereabouts plugin for secondary network configuration.
+	RegisterIPAMDriver(WhereaboutsIPAMType, &IPAMDelegator{pluginType: WhereaboutsIPAMType})
```

Comment:
Have you figured out why this is needed?

Reply:
I retested after removing the IPAM driver reg for whereabouts and with all the other fixes specific to whereabouts. It works good without registration. Not sure why we got the unsupport IPAM error earlier (before root cause other issues related to whereabouts configuration).

Thanks @jianjuns . I removed the IPAM reg part and pushed the updated changes.

With the updated changes, secondary interface is getting created successfully and got the IP assigned for the specific subnet (based on Annotation).
kubectl exec -it testpodsec15 -n kube-system -- ip addr show
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
3: eth0@if113: <BROADCAST,MULTICAST,UP,LOWER_UP,M-DOWN> mtu 1450 qdisc noqueue
    link/ether 7a:4c:50:d6:ea:24 brd ff:ff:ff:ff:ff:ff
    inet 172.25.0.46/24 brd 172.25.0.255 scope global eth0
       valid_lft forever preferred_lft forever
34: eth2: <NO-CARRIER,BROADCAST,MULTICAST,UP> mtu 1450 qdisc mq qlen 1000
    link/ether 5e:eb:3e:08:16:04 brd ff:ff:ff:ff:ff:ff
    inet 148.14.25.12/24 brd 148.14.25.255 scope global eth2
       valid_lft forever preferred_lft forever

Reply:
Sounds good!